src/HOHQMesh.jl

```diff
@@ -22,8 +22,12 @@ end
 # Main wrapper to generate a mesh from a control file
 export generate_mesh
 
+# Make the examples directory available to the user
+export examples_dir
+
 # Functions useful to demonstrate the interactive HQMTool
-export run_demo, ice_cream_cone_verbose_demo, ice_cream_cone_demo
+#export run_demo, ice_cream_cone_verbose_demo, ice_cream_cone_demo
```

Comment:
```suggestion
```

Reply:
This is still a WIP as I restructure the examples folder. This will get removed once I am finished. Also those functions will be removed entirely instead of just getting commented out.